hw2/04-background.html

```diff
@@ -19,10 +19,10 @@ <h1>Exercise 04 - Background</h1>
       </p>
       <div>
         <label>Background change interval (seconds): </label>
```

Comment:
To make this more accessible, the label should be connected to the input field with a 'for'.